### PR TITLE
Added support for Aventen S3 Sync

### PR DESCRIPTION
**Pull request by avencan**
## Description of Change
Introduced the Aventen-S3-Sync, an custom ESP32-S3 based board.

## Tests scenarios
The addition of the board has been tested with various different code and libraries, and we have ensured it will definitely work.

### CODE REVIEW

File: boards.txt [hunk 1]
```diff
@@ -1854,6 +1854,223 @@ tinypico.menu.EraseFlash.none.upload.erase_cmd=
...
+
+aventen_s3_sync.serial.disableDTR=false
+aventen_s3_sync.serial.disableRTS=false
+
+aventen_s3_sync.build.tarch=xtensa
+aventen_s3_sync.build.bootloader_addr=0x0
+aventen_s3_sync.build.target=esp32s3
+aventen_s3_sync.build.mcu=esp32s3
+aventen_s3_sync.build.core=esp32
+aventen_s3_sync.build.variant=esp32s3
```
Comment (P-R-O-C-H-Y) [inline boards.txt:1886]: You are not using your variants pins_arduino.h file. Please change this.
```suggestion
aventen_s3_sync.build.variant=Aventen_S3_Sync
```

File: boards.txt [hunk 2]
```diff
@@ -1854,6 +1854,223 @@ tinypico.menu.EraseFlash.none.upload.erase_cmd=
...
+aventen_s3_sync.build.board=ESP32S3_DEV
+
+aventen_s3_sync.build.usb_mode=1
+aventen_s3_sync.build.cdc_on_boot=0
+aventen_s3_sync.build.msc_on_boot=0
+aventen_s3_sync.build.dfu_on_boot=0
+aventen_s3_sync.build.f_cpu=240000000L
+aventen_s3_sync.build.flash_size=4MB
```
Comment (P-R-O-C-H-Y) [inline boards.txt:1894]: Idk what flash size you have on board. But here you have default 4MB, but later in config you have only 16 MB option.
So from that I assume this should be changed to 16MB if I am correct.
```suggestion
aventen_s3_sync.build.flash_size=16MB
```

File: boards.txt [hunk 3]
```diff
@@ -1854,6 +1854,223 @@ tinypico.menu.EraseFlash.none.upload.erase_cmd=
...
+aventen_s3_sync.menu.DebugLevel.debug.build.code_debug=4
+aventen_s3_sync.menu.DebugLevel.verbose=Verbose
+aventen_s3_sync.menu.DebugLevel.verbose.build.code_debug=5
+
+aventen_s3_sync.menu.EraseFlash.none=Disabled
+aventen_s3_sync.menu.EraseFlash.none.upload.erase_cmd=
+aventen_s3_sync.menu.EraseFlash.all=Enabled
+aventen_s3_sync.menu.EraseFlash.all.upload.erase_cmd=-e
+
+
```
Comment (P-R-O-C-H-Y) [inline boards.txt:2073]: Remove unnecessary line
```suggestion
```

File: boards.txt [hunk 4]
```diff
@@ -11520,7 +11737,7 @@ esp32vn-iot-uno.menu.EraseFlash.all.upload.erase_cmd=-e
 
 ##############################################################
 
-esp32doit-devkit-v1.name=DOIT ESP32 DEVKIT V1
+esp32doit-devkit-v1.name=DOIT ESP32 V1
```
Comment (P-R-O-C-H-Y) [inline boards.txt:11740]: Please revert this as this should not be part of the PR.

File: boards.txt [hunk 5]
```diff
@@ -24112,3 +24329,54 @@ nebulas3.menu.EraseFlash.all=Enabled
...
+nano_nora.build.partitions=app3M_fat9M_fact512k_16MB
+nano_nora.build.defines=-DBOARD_HAS_PIN_REMAP -DBOARD_HAS_PSRAM '-DUSB_MANUFACTURER="Arduino"' '-DUSB_PRODUCT="Nano ESP32"'
+nano_nora.build.loop_core=-DARDUINO_RUNNING_CORE=1
+nano_nora.build.event_core=-DARDUINO_EVENT_RUNNING_CORE=1
+nano_nora.build.psram_type=opi
+nano_nora.build.memory_type={build.boot}_{build.psram_type}
+
+nano_nora.tools.esptool_py.program.pattern_args=--chip {build.mcu} --port "{serial.port}" --before default_reset --after hard_reset write_flash -z --flash_mode {build.flash_mode} --flash_freq {build.flash_freq} --flash_size {build.flash_size} {build.bootloader_addr} "{build.path}/{build.project_name}.bootloader.bin" 0x8000 "{build.path}/{build.project_name}.partitions.bin" 0xe000 "{runtime.platform.path}/tools/partitions/boot_app0.bin" 0xf70000 "{build.variant.path}/extra/nora_recovery/nora_recovery.ino.bin" 0x10000 "{build.path}/{build.project_name}.bin"
+
+##############################################################
```
Comment (P-R-O-C-H-Y) [inline boards.txt:24382]: This should not be there. Not part of this PR.

**Review by P-R-O-C-H-Y — CHANGES_REQUESTED** [review]
@avencan Please take a look on my comments.

**Comment by avencan** [comment]
Hi @P-R-O-C-H-Y 

I have made the changes as requested, aswell as removing some unnecessary things, I hope everything is fine with it now :).

Here's some details about the board - if needed for errors.

image
image

Thanks

**Comment by P-R-O-C-H-Y** [comment]
Hi @avencan,

the new commit with changes to boards.txt file is messed up: `24,329 additions, 24,382 deletions` so I cannot easily check the changes. Can you try to revert this commit and push the changes again? Thanks

**Comment by avencan** [comment]
Hi @P-R-O-C-H-Y 

I don't think I can revert it, and I don't really know why this happened, would you be able to check the changes on a local git client? Sorry about this

Thank you

**Comment by P-R-O-C-H-Y** [comment]
Hi @avencan, i will try if that will do good, but first can you please resolve conflicts please? Thanks

**Pull request closed without merging**